.changeset/format-changelogs.cjs

```diff
@@ -38,11 +38,17 @@ async function getDependencyReleaseLine(changesets, dependenciesUpdated) {
 }
 
 async function getReleaseLine(changeset) {
-  const { contributor, link, summary } = await extractInformation(changeset);
+  try {
+    const { contributor, link, summary } = await extractInformation(changeset);
```

Comment:
At the moment, `@changeset/get-github-info` is rather buggy and can lead to runtime errors:

```sh
ERROR: getReleaseLine (Cannot read properties of null (reading 'author'))
```